project.yml

```
@@ -34,7 +34,9 @@ targets:
     platform: iOS
     sources:
       - UhooiPicBook
-      - Shared
+      - path: Shared
+        excludes: 
+          -  "IntentDefinition/**"
```

製品ターゲットにも `IntentDefinition` は必要とのことなので、除外は不要です。

```suggestion
      - Shared
```